test/io-tests.sh

```diff
@@ -231,8 +234,10 @@ readSecretFromFile ascii.b64 'Base64 (ASCII)'
 readSecretFromFile utf8.b64 'Base64 (UTF-8)'
 readSecretFromFile binary.b64 'Base64 (binary)'
 
-readDbUriFromFile uri.noeol "(no EOL)"
-readDbUriFromFile uri.txt "(EOL)"
+eol=$'\x0a'
```

Comment:
Nice. Thanks for taking the time to refactor these tests! Comments are really good too.